### DISCUSSION TITLE
blog/2019/01/29/diff-means-half-dozen-ways/index

### DISCUSSION THREAD

**Discussion by giscus** [andrewheiss/ath-quarto#58] [Blog comments]
# blog/2019/01/29/diff-means-half-dozen-ways/index

Learn how to run standard t-tests, simulations, and Bayesian difference in means tests with R and Stan

https://www.andrewheiss.com/blog/2019/01/29/diff-means-half-dozen-ways/index.html

**Comment by arracadas** [comment]
excellent! thanks for sharing,  I am new to bayesian MCMC, but this is a great comparison of methods.
